### PR TITLE
PSP-5715 : Sec3/Sec6 add forms to acquisition file

### PR DESCRIPTION
**Pull request by eddherrera**
image

### CODE REVIEW

File: source/backend/api/Services/CompensationRequisitionService.cs [hunk 1]
```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Security.Claims;
+using DocumentFormat.OpenXml.Office2010.Excel;
```
Comment (asanchezr) [inline source/backend/api/Services/CompensationRequisitionService.cs:5]: This needs to be removed (seems like you are sharing the same branch from your last pr?)

File: source/frontend/src/features/mapSideBar/acquisition/tabs/AcquisitionFileTabs.test.tsx [hunk 2]
```diff
@@ -78,4 +78,51 @@ describe('AcquisitionFileTabs component', () => {
       expect(getByText('Documents')).toHaveClass('active');
     });
   });
+
+  it('hides the expropiation tab when the Acquisition file type is "Consensual Agreement"', () => {
```
Comment (asanchezr) [inline source/frontend/src/features/mapSideBar/acquisition/tabs/AcquisitionFileTabs.test.tsx:82]: please replace all instances of "expropiation" with "expropriation". I see it in comments like this and also on file names, section names, etc. 
Thanks!

**Review by asanchezr — APPROVED** [review]

File: source/backend/api/Services/CompensationRequisitionService.cs (hunk 1 above)
Comment (eddherrera) [reply source/backend/api/Services/CompensationRequisitionService.cs]: removed

File: source/frontend/src/features/mapSideBar/acquisition/tabs/AcquisitionFileTabs.test.tsx (hunk 2 above)
Comment (eddherrera) [reply source/frontend/src/features/mapSideBar/acquisition/tabs/AcquisitionFileTabs.test.tsx]: Updated

**Pull request merged**